src/component/footer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const date= Date();
+
+function Footer(){
+    return(
```

Comment:
why is date outside of the component? if you would like you could directly use the `Date()` constructor within the `jsx` or place it inside the component for better performance
```suggestion
function Footer(){
    const date = Date();
    return(
```